### ISSUE TITLE
HistoCAT crashes when trying to run tSNE or PCA (AMD based CPU)

### ISSUE THREAD

**Issue by jparkinson375** [BodenmillerGroup/histoCAT#51]
I have an issue when I try to run tSNE or PCA embedding on my data in Windows 10. This used to run fine on my other machine running Windows 10 and has only started recently. The only change I can think of is that I have moved from an Intel-based system to a new AMD-based one.

After a bit of googling, I found the following topics on an error that occurs in MATLAB with old AMD CPUs and I am wondering if this could be causing the issue?

[https://uk.mathworks.com/matlabcentral/answers/375023-why-my-matlab-has-encountered-with-an-internal-problem-ans-needs-to-close-when-i-tried-to-run-a-code#answer_298203](url)

[https://uk.mathworks.com/matlabcentral/answers/138692-matrix-multiply-crashes-matlab-2014a-on-a-amd-a6-1450-apu-under-either-windows-8-1-64-bit-or-linux](url)

The data loads into histoCAT fine, I can view the images and can even run Phenograph without an issues. However, as soon as I try to run tSNE or PCA it loads for a second and brings up the loading bar before crashing to desktop.

When I run the program in command line I get the following error:

------------------------------------------------------------------------
        Illegal instruction detected at Mon Mar 30 14:23:43 2020
------------------------------------------------------------------------

Configuration:
  Crash Decoding     : Disabled
  Default Encoding   : windows-1252
  Graphics card 1    : NVIDIA ( 0x10de ) NVIDIA GeForce GTX 1080 Version 26.21.14.4575
  MATLAB Architecture: win64
  MATLAB Root        : C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84
  MATLAB Version     : 8.4.0.150421 (R2014b)
  Operating System   : Microsoft Windows 10 Pro
  Processor ID       : x86 Family 143 Model 113 Stepping 0, AuthenticAMD
  Software OpenGL    : 0
  Virtual Machine    : Java 1.7.0_11-b21 with Oracle Corporation Java HotSpot(TM) 64-Bit Server VM mixed mode
  Window System      : Version 10.0 (Build 18362)

Fault Count: 1

Abnormal termination:
Illegal instruction

Register State (from fault):
  RAX = 000000005f7e8000  RBX = 000000005f7e9000
  RCX = 0000000023b348b0  RDX = 0000000145db2bb0
  RSP = 0000000023b346c0  RBP = 0000000023b356b8
  RSI = 00000000c5098860  RDI = 0000000023b36008

   R8 = 0000000000000240   R9 = 000000005f7e9000
  R10 = 0000000000000360  R11 = 0000000000000120
  R12 = 0000000000000012  R13 = 000000000000000c
  R14 = 0000000000000020  R15 = 0000000000000012

  RIP = 00000000c3f1a342  EFL = 00010212

   CS = 0033   FS = 0053   GS = 002b

Stack Trace (from fault):
[  0] 0x00000000c3f1a342 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+26387266 xerbla+23942290
[  1] 0x00000000c34939ea C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+15350250 xerbla+12905274
[  2] 0x00000000c3481ec1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+15277761 xerbla+12832785
[  3] 0x00000000c26df9c2 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+00981442 mkl_cbwr_set+00610690
[  4] 0x00000000c26db35c C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+00963420 mkl_cbwr_set+00592668
[  5] 0x00000000c29fa40b C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+04236299 xerbla+01791323
[  6] 0x00000000c29f8c58 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+04230232 xerbla+01785256
[  7] 0x00000000c26f15b8 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+01054136 mkl_cbwr_set+00683384
[  8] 0x00000000c2a9b0f2 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+04894962 xerbla+02449986
[  9] 0x00000000c2aa8297 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+04948631 xerbla+02503655
[ 10] 0x00000000c2619cbc C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mkl.dll+00171196 dsyevd+00000236
[ 11] 0x00000000ee226f00 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwmathlinalg.dll+00552704 UMFPACK_RealLUFactor::whackSymbolic+00028848
[ 12] 0x00000000ee227a86 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwmathlinalg.dll+00555654 UMFPACK_RealLUFactor::whackSymbolic+00031798
[ 13] 0x00000000ee1b9ba8 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwmathlinalg.dll+00105384
[ 14] 0x00000000ee1ba3e9 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwmathlinalg.dll+00107497
[ 15] 0x000000001d8e0fce C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00069582 mdLogging::log_function_exit+00000110
[ 16] 0x000000001d8e09cc C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068044 Mfunction_handle::dispatch+00000476
[ 17] 0x000000001cf09d7c C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00236924 inGetSeqVarPtrsAndStaticLevelFromCurrentWS+00071596
[ 18] 0x000000001cf47d06 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00490758 inPathNotification::function_delete_notification+00179638
[ 19] 0x000000001cee8cfa C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00101626 MathWorks::MException::IMExceptionData::IMExceptionData+00026586
[ 20] 0x000000001cf3b708 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00440072 inPathNotification::function_delete_notification+00128952
[ 21] 0x000000001cf13ea5 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00278181 inGetNameInSymbolTable+00022053
[ 22] 0x000000001cf11b4f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00269135 inGetNameInSymbolTable+00013007
[ 23] 0x000000001cf11ab1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00268977 inGetNameInSymbolTable+00012849
[ 24] 0x000000001cf3406f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00409711 inPathNotification::function_delete_notification+00098591
[ 25] 0x000000001cf327af C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00403375 inPathNotification::function_delete_notification+00092255
[ 26] 0x000000001cf31359 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00398169 inPathNotification::function_delete_notification+00087049
[ 27] 0x000000001d8e05e3 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00067043 Mfh_file::dispatch_fh+00000659
[ 28] 0x000000001d8e0aee C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068334 Mfunction_handle::dispatch+00000766
[ 29] 0x000000001ceeb606 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00112134 MathWorks::MException::IMExceptionData::IMExceptionData+00037094
[ 30] 0x000000001ceeaa20 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00109088 MathWorks::MException::IMExceptionData::IMExceptionData+00034048
[ 31] 0x000000001cee9219 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00102937 MathWorks::MException::IMExceptionData::IMExceptionData+00027897
[ 32] 0x000000001cee8de5 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00101861 MathWorks::MException::IMExceptionData::IMExceptionData+00026821
[ 33] 0x000000001cf3b708 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00440072 inPathNotification::function_delete_notification+00128952
[ 34] 0x000000001cf13ea5 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00278181 inGetNameInSymbolTable+00022053
[ 35] 0x000000001cf11b4f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00269135 inGetNameInSymbolTable+00013007
[ 36] 0x000000001cf11ab1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00268977 inGetNameInSymbolTable+00012849
[ 37] 0x000000001cf3406f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00409711 inPathNotification::function_delete_notification+00098591
[ 38] 0x000000001cf327af C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00403375 inPathNotification::function_delete_notification+00092255
[ 39] 0x000000001cf31359 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00398169 inPathNotification::function_delete_notification+00087049
[ 40] 0x000000001d8e05e3 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00067043 Mfh_file::dispatch_fh+00000659
[ 41] 0x000000001d8e0aee C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068334 Mfunction_handle::dispatch+00000766
[ 42] 0x000000001ceeb606 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00112134 MathWorks::MException::IMExceptionData::IMExceptionData+00037094
[ 43] 0x000000001ceeaa20 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00109088 MathWorks::MException::IMExceptionData::IMExceptionData+00034048
[ 44] 0x000000001cee9219 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00102937 MathWorks::MException::IMExceptionData::IMExceptionData+00027897
[ 45] 0x000000001cee8de5 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00101861 MathWorks::MException::IMExceptionData::IMExceptionData+00026821
[ 46] 0x000000001cf3b708 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00440072 inPathNotification::function_delete_notification+00128952
[ 47] 0x000000001cf13ea5 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00278181 inGetNameInSymbolTable+00022053
[ 48] 0x000000001cf11b4f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00269135 inGetNameInSymbolTable+00013007
[ 49] 0x000000001cf11ab1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00268977 inGetNameInSymbolTable+00012849
[ 50] 0x000000001cf3406f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00409711 inPathNotification::function_delete_notification+00098591
[ 51] 0x000000001cf327af C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00403375 inPathNotification::function_delete_notification+00092255
[ 52] 0x000000001cf31359 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00398169 inPathNotification::function_delete_notification+00087049
[ 53] 0x000000001d8e05e3 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00067043 Mfh_file::dispatch_fh+00000659
[ 54] 0x000000001d8e0aee C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068334 Mfunction_handle::dispatch+00000766
[ 55] 0x000000001d8df870 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00063600 mdSetCurrentIntFcnIndexProtected::mdSetCurrentIntFcnIndexProtected+00000816
[ 56] 0x00000000fe165044 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmx.dll+00413764 matrix::detail::noninlined::mx_array_api::mxFevalFunctionHandle+00000036
[ 57] 0x000000001cfd372b C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+01062699 inFullFevalFcn+00000139
[ 58] 0x000000001d8e0fce C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00069582 mdLogging::log_function_exit+00000110
[ 59] 0x000000001d8e09cc C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068044 Mfunction_handle::dispatch+00000476
[ 60] 0x000000001cf12b2f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00273199 inGetNameInSymbolTable+00017071
[ 61] 0x000000001cf140c0 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00278720 inGetNameInSymbolTable+00022592
[ 62] 0x000000001cf11b4f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00269135 inGetNameInSymbolTable+00013007
[ 63] 0x000000001cf11ab1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00268977 inGetNameInSymbolTable+00012849
[ 64] 0x000000001cf3406f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00409711 inPathNotification::function_delete_notification+00098591
[ 65] 0x000000001cf327af C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00403375 inPathNotification::function_delete_notification+00092255
[ 66] 0x000000001cf31359 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00398169 inPathNotification::function_delete_notification+00087049
[ 67] 0x000000001d8e05e3 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00067043 Mfh_file::dispatch_fh+00000659
[ 68] 0x000000001d8e0aee C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068334 Mfunction_handle::dispatch+00000766
[ 69] 0x000000001cf12b2f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00273199 inGetNameInSymbolTable+00017071
[ 70] 0x000000001cf140c0 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00278720 inGetNameInSymbolTable+00022592
[ 71] 0x000000001cf11b4f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00269135 inGetNameInSymbolTable+00013007
[ 72] 0x000000001cf11ab1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00268977 inGetNameInSymbolTable+00012849
[ 73] 0x000000001cf3406f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00409711 inPathNotification::function_delete_notification+00098591
[ 74] 0x000000001cf327af C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00403375 inPathNotification::function_delete_notification+00092255
[ 75] 0x000000001cf31359 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00398169 inPathNotification::function_delete_notification+00087049
[ 76] 0x000000001d8e05e3 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00067043 Mfh_file::dispatch_fh+00000659
[ 77] 0x000000001d8e0aee C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00068334 Mfunction_handle::dispatch+00000766
[ 78] 0x000000001cf12b2f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00273199 inGetNameInSymbolTable+00017071
[ 79] 0x000000001cf140c0 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00278720 inGetNameInSymbolTable+00022592
[ 80] 0x000000001cf11b4f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00269135 inGetNameInSymbolTable+00013007
[ 81] 0x000000001cf11ab1 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00268977 inGetNameInSymbolTable+00012849
[ 82] 0x000000001cf3406f C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00409711 inPathNotification::function_delete_notification+00098591
[ 83] 0x000000001cf327af C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+00403375 inPathNotification::function_delete_notification+00092255
[ 84] 0x000000001d8e49df C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00084447 Mfh_file::dispatch_fh+00000335
[ 85] 0x000000001d8e58c7 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00088263 mdIsClear+00000343
[ 86] 0x000000001d8df870 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_dispatcher.dll+00063600 mdSetCurrentIntFcnIndexProtected::mdSetCurrentIntFcnIndexProtected+00000816
[ 87] 0x00000000fe165044 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmx.dll+00413764 matrix::detail::noninlined::mx_array_api::mxFevalFunctionHandle+00000036
[ 88] 0x000000001cfd372b C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\m_interpreter.dll+01062699 inFullFevalFcn+00000139
[ 89] 0x00000000fb3e084b C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwgui.dll+00067659 MCallback::_execute+00000203
[ 90] 0x00000000fb3e0eed C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwgui.dll+00069357 MCallback::execute+00000413
[ 91] 0x000000009e3ad6fc C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hgdatatypes.dll+00579324 Java_com_mathworks_test_hg_types_UTF16StringNativeTester_set+00007708
[ 92] 0x000000009e3ac8e7 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hgdatatypes.dll+00575719 Java_com_mathworks_test_hg_types_UTF16StringNativeTester_set+00004103
[ 93] 0x000000009e3ad253 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hgdatatypes.dll+00578131 Java_com_mathworks_test_hg_types_UTF16StringNativeTester_set+00006515
[ 94] 0x000000009e33117d C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hgdatatypes.dll+00070013 graphics::datatype::CallbackDataType::destroy+00001373
[ 95] 0x000000001da1b722 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mcos.dll+00440098 mcos::COSMethodInfo::isDestructible+00000706
[ 96] 0x000000001da1b68a C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mcos.dll+00439946 mcos::COSMethodInfo::isDestructible+00000554
[ 97] 0x000000001d9d94a7 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mcos.dll+00169127 mcos::COSInterfacePtr::isNULL+00000615
[ 98] 0x000000007f6712ee C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+05247726 gbtdatatypes::SingleCharacterDataType::getType+01432430
[ 99] 0x000000007f29cf1e C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+01232670 UIClientComponentInterface::requestFocus+00427214
[100] 0x000000007f29f32c C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+01241900 UIClientComponentInterface::requestFocus+00436444
[101] 0x000000007f29f4ad C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+01242285 UIClientComponentInterface::requestFocus+00436829
[102] 0x000000007f3135a3 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+01717667 UIClientComponentInterface::requestFocus+00912211
[103] 0x000000007f28a1be C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+01155518 UIClientComponentInterface::requestFocus+00350062
[104] 0x000000007f3e7eac C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+02588332 hgImage+00036796
[105] 0x000000007f3e2839 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\hg.dll+02566201 hgImage+00014665
[106] 0x000000001dcbccb8 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\udd.dll+00249016 UDListener::removeListenerContainer+00000648
[107] 0x000000001dcb571c C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\udd.dll+00218908 UDEventInfo::send+00000092
[108] 0x00000000168b1008 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\udd_mi.dll+00069640 MErrorException::operator=+00011144
[109] 0x00000000fab5d3a6 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\uiw.dll+00512934 UIW_AttachThreadInput+00001270
[110] 0x00000000fab5cc35 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\uiw.dll+00511029 wsd_to_MSW+00004373
[111] 0x00000000fab5ccb9 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\uiw.dll+00511161 wsd_to_MSW+00004505
[112] 0x00007ffb74a510b5                     C:\WINDOWS\System32\USER32.dll+00528565 GetClassNameA+00000581
[113] 0x00007ffb749f20a9                     C:\WINDOWS\System32\USER32.dll+00139433 IsWindowVisible+00000729
[114] 0x00007ffb749f2035                     C:\WINDOWS\System32\USER32.dll+00139317 IsWindowVisible+00000613
[115] 0x00007ffb7551fe34                      C:\WINDOWS\SYSTEM32\ntdll.dll+00654900 KiUserCallbackDispatcher+00000036
[116] 0x00007ffb734d10c4                     C:\WINDOWS\System32\win32u.dll+00004292 NtUserPeekMessage+00000020
[117] 0x00007ffb749e94e2                     C:\WINDOWS\System32\USER32.dll+00103650 PeekMessageW+00000482
[118] 0x00007ffb749e9ad9                     C:\WINDOWS\System32\USER32.dll+00105177 PeekMessageA+00000105
[119] 0x00000000fab0a26b C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\uiw.dll+00172651 UIW_ShowMenuItem+00003995
[120] 0x00000000fab5dce2 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\uiw.dll+00515298 UIW_SuspendAttachThreadInput+00000466
[121] 0x0000000100118223 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwservices.dll+01212963 services::system_events::PpeDispatchHook::dispatchOne+00032291
[122] 0x0000000100123c95 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwservices.dll+01260693 sysq::addProcessPendingEventsUnitTestHook+00005813
[123] 0x0000000100123d50 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwservices.dll+01260880 sysq::addProcessPendingEventsUnitTestHook+00006000
[124] 0x0000000100125b75 C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwservices.dll+01268597 sysq::getCondition+00004629
[125] 0x00000001001274ba C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\libmwservices.dll+01275066 svWS_ProcessPendingEvents+00000138
[126] 0x00000000fd0a783d C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mcr.dll+00161853 mcr::runtime::setInterpreterThreadToCurrent+00023677
[127] 0x00000000fd0a84aa C:\Program Files\MATLAB\MATLAB Compiler Runtime\v84\bin\win64\mcr.dll+00165034 mcr::runtime::setInterpreterThreadToCurrent+00026858

**Comment by DenisSch** [comment]
Thank you for the detailed issue description. Are you able to run histoCAT from source in MATLAB? Do you get a similar issue?

**Comment by jparkinson375** [comment]
I just tried running it from source in MATLAB and it ran perfectly, no issue with Tsne or PCA. 

Just for sanity, I ran it from the Windows application again and it crashed to the desktop in the same manner as before.

Luckily I have a MATLAB license through my institution so I can run it from source to fix the problem I'm not sure if it will be an issue for others with AMD hardware, who may not have access to a license.

Thank you for the help and all the work you do on the software, it is brilliant.
